src/components/structures/MatrixChat.js

```diff
@@ -1304,9 +1304,9 @@ export default React.createClass({
             }
         });
 
-        const dft = new DecryptionFailureTracker((failure) => {
+        const dft = new DecryptionFailureTracker((total) => {
             // TODO: Pass reason for failure as third argument to trackEvent
```

Comment:
This TODO is now a bit spurious, or at least outdated, as there would be multiple failure reasons

Reply:
It's still valid though, that's exactly the change we would need. The change would be making `total` a map of error types to cardinality.